recipes/xclip/COPYING

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE

```

Comment:
It looks like this should be in the tarball. If so, please remove this from the recipe submission.